src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/repository.py

```diff
@@ -130,9 +146,43 @@ def _obtain_data_from_registry(login_server,
     return resultList
 
 
+def _get_manifest_digest(login_server, path, username, password, **kwargs):
+    retry_times = kwargs['retry_times'] if 'retry_times' in kwargs else 3
+    retry_interval = kwargs['retry_interval'] if 'retry_interval' in kwargs else 5
+
+    for i in range(0, retry_times):
+        try:
+            errorMessage = None
+            response = requests.get(
```

Comment:
Is this sending a manifest GET request? If you only want to know the digest from response header, you can send HEAD request to avoid transferring the body which you don't care.